src/core/picker.cpp

```diff
@@ -82,16 +82,38 @@ void Picker::Show(PickerInfo* Picker, Color* Colors, uint32 BlankTexture) // TOD
     {
         ImGui::BeginChild("HSV Gradients", Vec2(315, 258));
         ImGui::EndChild();
-        int32 col[3];
-        col[0] = (int)(Picker->NewColor.r * 255.0f);
-        col[1] = (int)(Picker->NewColor.g * 255.0f);
-        col[2] = (int)(Picker->NewColor.b * 255.0f);
-        if (ImGui::InputInt3("RGB", col))
+        int rgb_color[3];
+        rgb_color[0] = (int)(Picker->NewColor.r * 255.0f);
+        rgb_color[1] = (int)(Picker->NewColor.g * 255.0f);
+        rgb_color[2] = (int)(Picker->NewColor.b * 255.0f);
+        if (ImGui::InputInt3("RGB", rgb_color))
         {
-            Picker->NewColor = Color(col[0], col[1], col[2]); // TODO: Clamping
+			int r = Math::Clamp(rgb_color[0], 0, 255);
+			int g = Math::Clamp(rgb_color[1], 0, 255);
+			int b = Math::Clamp(rgb_color[2], 0, 255);
+            Picker->NewColor = Color(r, g, b);
             Math::RGBtoHSV(Picker->NewColor.r, Picker->NewColor.g, Picker->NewColor.b,
                     Picker->CursorH, Picker->CursorSV.x, Picker->CursorSV.y);
         }
+		char hex_color[6 + 1]; // null-terminated
```

Comment:
According to local variable convention, this variable should be named hexColor.

(I know I'm deviating from the convention myself in a few places, especially in terms of the capitalization, but I'll be whipping the code into shape soon.)